### DISCUSSION TITLE
Licensing questions

### DISCUSSION THREAD

**Discussion by draconius1** [naturalcrit/homebrewery#4483] [Q&A]
I really appreciate your work and thanks for what you do for the community. I would like to publish some content on DMSGuild and I know that you have given permission to use your program for publishing. My question is the background , letter fonts, boxed graphics, footer art, that are default with the program royalty free or included with the license you give to use your site?

**Comment by athelu** [comment]
I believe that technically the look and feel of the default document is modeled after the WOTC Trade-dress - so the author here has no ability to provide a license.

Any content generated would have to be seen as free community content and published under OGL with no profit since it is relying on the trade-dress.

**Comment by Gazook89** [comment]
@calculuschild @ericscheid This can probably be closed, unless something is going to be added to the site to clarify licensing.

**Comment by calculuschild** [comment]
The closest thing we've got is this on the homepage:

> Legal Junk
> The Homebrewery is licensed using the MIT License. Which means you are free to use The Homebrewery is any way that you > want, except for claiming that you made it yourself.
> 
> If you wish to sell or in some way gain profit for what's created on this site, it's your responsibility to ensure you have the > 
> proper licenses/rights for any images or resources used.

All of the graphics/fonts were created by us or by users on reddit (/r/unearthedarcana, etc.), so the fonts and graphics individually are fair use. The DMSGuild even provides [style guides](https://www.dmsguild.com/product/267467/DMs-Guild-Creator-Resource--Style-Guide-Resources?src=newest) and artwork for authors with example fonts, etc. to use that very closely match the published WOTC styles, minus coloring and background images, so it would seem if you are aiming to publish on DMSGuild you have more liberty to use their Trade Dress (although I can't find anything specific on that). Publishing your work elsewhere for profit is probably not so obvious.

I'm going to opt to keep this open for now, because this is something we probably should nail down a little better with more research.

**Comment by Gazook89** [comment]
I randomly came across [this Angry DM post](https://theangrygm.com/save-vs-lawsuit-how-to-publish-dd-content/) from 5 years ago that in my opinion provides some plain english clarity about the WotC IP environment.  Just wanted to leave it here in case others coming across this Issue find it informative.

**Comment by dbolack-ab** [comment]
Can we find providence on the various assets so we can provide clear permission?

**Comment by athelu** [comment]
> Can we find providence on the various assets so we can provide clear permission?

It is the Trade Dress that is the issue. Trade dress is the LOOK of something. That is considered part of a product. Homebrewery (by default) looks almost exactly like WOTC trade dress. The fonts are not the exact same, the background is not the exact same, but it was all built with the intent to be as close as possible while remaining free to use. This means products created with the default homebrewery settings violate WOTC trade dress UNLESS you are making something for personal use or to publish on DMSGuild. 

It might be simple as much as stating just that:
Default homebrewery style documents are for personal use or publishing to DMSguild. 

And it is very much possible to over-ride the styling to mimic many other products. I peronally have created styles to duplicate Zombicide and Wyrd: Through the Breach

**Comment by dbolack-ab** [comment]
> It is the Trade Dress that is the issue. Trade dress is the LOOK of something. That is considered part of a product. Homebrewery (by default) looks almost exactly like WOTC trade dress. The fonts are not the exact same, the background is not the exact same, but it was all built with the intent to be as close as possible while remaining free to use. This means products created with the default homebrewery settings violate WOTC trade dress UNLESS you are making something for personal use or to publish on DMSGuild.

These are two distinctly different concerns. 

The first concern is do we have authorization to use and distribute for downstream use the various assets in the project.   Based on my cursory looks at different bits the answer is murky.

The second is about if the trade dress is enforceable ( most likely ) and if they care to. We have statements from Wotc reps on direct questioning that include the response: ( #3728 )

> -There's no need to credit WotC with the design as trade dress isn't trademarked.

For the rest of this post, I am going to assume that the second is true - after all, we can defend the case even if it isn't enforceable.

> It might be simple as much as stating just that: Default homebrewery style documents are for personal use or publishing to DMSguild.

This is not a defense for our project. It would be argued we are functionally conspiring for the infringement actions.

**Comment by 5e-Cleric** [comment]
Moved to a discussion, which is the form for this
